integration/resources/templates/combination/connector_appsync_to_table.yaml

```diff
@@ -0,0 +1,169 @@
+Resources:
+  NotesTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: notes-table
+      AttributeDefinitions:
+      - AttributeName: UserId
+        AttributeType: S
+      KeySchema:
+      - AttributeName: NoteId
+        KeyType: HASH
+
+  DynamoDBRole:
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: appsync-dynamodb-role
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+        - Effect: Allow
+          Action:
+          - sts:AssumeRole
+          Principal:
+            Service:
+            - appsync.amazonaws.com
+
+  AppSyncApi:
+    Type: AWS::AppSync::GraphQLApi
+    Properties:
+      AuthenticationType: API_KEY
+      Name: AppSyncApi
+
+  ApiKey:
+    Type: AWS::AppSync::ApiKey
+    Properties:
+      ApiId: !GetAtt AppSyncApi.ApiId
+
+  ApiSchema:
+    Type: AWS::AppSync::GraphQLSchema
+    Properties:
+      ApiId: !GetAtt AppSyncApi.ApiId
+      Definition: |
+        type Note {
+          NoteId: ID!
+          title: String
+          content: String
+        }
+        type Query {
+          getNote(NoteId: ID!): Note
+        }
+        type Mutation {
+          saveNote(NoteId: ID!, title: String!, content: String!): Note!
+        }
+        type Schema {
+          query: Query
+          mutation: Mutation
+        }
+
+  NotesTableDataSource:
+    Type: AWS::AppSync::DataSource
+    Properties:
+      ApiId: !GetAtt AppSyncApi.ApiId
+      Name: NotesTableDataSource
+      Type: AMAZON_DYNAMODB
+      ServiceRoleArn: !GetAtt DynamoDBRole.Arn
+      DynamoDBConfig:
+        TableName: !Ref NotesTable
+        AwsRegion: !Sub ${AWS::Region}
+
+  DataSourceToTableConnector:
+    Type: AWS::Serverless::Connector
+    Properties:
+      Source:
+        Id: NotesTableDataSource
+      Destination:
+        Id: NotesTable
+      Permissions:
+      - Read
+      - Write
+
+  TriggerFunction:
+    Type: AWS::Serverless::Function
+    Properties:
+      Environment:
+        Variables:
+          API_KEY: !GetAtt ApiKey.ApiKey
+          GRAPHQL_URL: !GetAtt AppSyncApi.GraphQLUrl
+      Runtime: nodejs18.x
+      Handler: index.handler
+      InlineCode: |
```

Comment:
Does this test that it's able to retrieve the saved value? Currently it seems it's only checking the status code, but not the actual output.

Reply:
Lambda returns retrieved value in `Body` but universal test (python code) check only errors. I obviously ran this lambda separately to make sure it works

Reply:
I've changed the function to check response body even if response doesn't have `errors` field. The function still returns status which is either 200 or 400 for the sake of general test which calls trigger function. Since I set trigger function response status value manually based on response body analysis instead of using AppSync response status, there is no danger that status will be 200 even with errors in the body